jobs/JEVS_MESOSCALE_STATS

```diff
@@ -71,12 +71,13 @@ if [ $MODELNAME = nam ] || [ $MODELNAME = rap ]; then
   export EVSINspcotlk=${EVSINspcotlk:-$COMIN/prep/cam}
   export DCOMINmrms=${DCOMINmrms:-$DCOMROOT/ldmdata/obs/upperair/mrms}
   export DCOMINnohrsc=${DCOMINnohrsc:-$DCOMROOT}
+  export EVSIN=${COMIN:-$(compath.py $envir/com/$NET/$evs_ver)}
 
   if [ $VERIF_CASE = snowfall ]; then
       np=$(date +"%H");         nq=$(($np%6));        nr=$(($np-$nq));
       printf -v ns "%02d" $np;        printf -v nt "%02d" $nr;
       export cyc=$nt;
-      export cycle=t${cyc}z
+      export cycle=t${vhr}z
   fi
```

Comment:
@RoshanShrestha-NOAA , let's work on this area. NCO does not want us to use the unix date command. Let's modify lines 76-81 to look like this. See if this will be OK.
```
  if [ $VERIF_CASE = snowfall ]; then
      np=$vhr;         nq=$(($np%6));        nr=$(($np-$nq));
      printf -v ns "%02d" $np;        printf -v nt "%02d" $nr;
      export vhr=$nt;
      export cycle=t${vhr}z
  fi
```
I think setting np to $vhr is OK since the job is run specifically on 00, 06, 12, and 18 Z for snowfall stats.

Reply:
Sure, I will correct it. Thanks.